keps/sig-node/4216-image-pull-per-runtime-class/README.md

```diff
@@ -0,0 +1,611 @@
+    of a node? `No`
+
+###### Does enabling the feature change any default behavior?
+
+No. Aim of this KEP to ensure default behavior is not changed. Any changes to CRI and kubelet will be under feature gate as mentioned in earlier sections.
+
+###### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement)?
+
+Yes. All the code changes needed will be under feature gate `RuntimeClassInCriCalls` that will be off by default for alpha.
+Default exisiting behavior of kubelet will be ensured.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+It should be a no-op
+
+###### Are there any tests for feature enablement/disablement?
+
+Yes, this test will be added for alpha.
+
+### Monitoring Requirements
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+`Note:` This section will be filled once we move to beta.
+
+###### How can an operator determine if the feature is in use by workloads?
+
+Image pulls for non-default runtime class goes through. Alternately, one could check the status of the feature gate.
+
+###### How can someone using this feature know that it is working for their instance?
+
+Image pull goes through successfully when non-default runtimehandler is used.
+For example, if host is a windows node, user should be able to pull images for hyperV isolation even if the container image OSVersion does not match the host OS version.
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+Exisiting behavior of kubelet should not change when runtime class is not speified when image is pulled.
+
+<!--
+This is your opportunity to define what "normal" quality of service looks like
+for a feature.
+
+It's impossible to provide comprehensive guidance, but at the very
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+`N/A`
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+`N/A`
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+
+### Dependencies
+
+<!--
+This section must be completed when targeting beta to a release.
+-->
+
+###### Does this feature depend on any specific services running in the cluster?
+
+kubelet
```

Comment:
No need for this. The spirit of this question is to identify dependencies - for example, an autoscaler likely requires a metrics service.